lldb/test/API/commands/session/save/TestSessionSave.py

```diff
@@ -61,8 +61,14 @@ def test_session_save(self):
         self.assertFalse(res.Succeeded())
         raw += self.raw_transcript_builder(cmd, res)
 
-        tf = tempfile.NamedTemporaryFile()
-        output_file = tf.name
+        fd, output_file = tempfile.mkstemp()
```

Comment:
I'd just put the file into the build directory of the test (`self.getBuildArtifact("my_session")`). Then you don't need to worry about cleanups or collisions, as that directory is nuked before running the test.